IntroToCSharp/Shared/Components/Activities/SubSection.razor

```diff
@@ -0,0 +1,42 @@
+@inject NavigationManager NavigationManager
+@inject Blazored.LocalStorage.ILocalStorageService localStorage
+
+<MudExpansionPanel @bind-IsExpanded="@IsExpanded" onclick="@(new Action(OnExpand))">
+    <TitleContent>
+        <h3 class="@_theme pa-2">@Title</h3>
```

Comment:
The only difference between Section and SubSection is this `h2` / `h3` tag. The rest of the surrounding code is copy / pasted. You should instead make this a parameter of the Section class. For example, something like this:

```
<Section IsSubsection="h3">
</Section>
```

Thoughts?